dvc/repo/brancher.py

```diff
@@ -24,21 +29,22 @@ def brancher(  # noqa: E302
 
     scm = self.scm
 
-    if self.scm.is_dirty():
-        from dvc.scm.tree import WorkingTree
-
-        self.tree = WorkingTree(self.root_dir)
+    is_dirty = scm.is_dirty()
+    self.tree = WorkingTree(self.root_dir)
+    if is_dirty:
         yield "Working Tree"
```

Comment:
Let's rename to "working tree", as requested by https://github.com/iterative/dvc/issues/2079#issuecomment-497967226